app/assets/javascripts/admin/views/edition_workflow/confirm_unpublish.js

```diff
@@ -8,9 +8,11 @@
 
       this.revealCorrectForm()
       this.hideExplanationIfRedirecting()
+      this.revealNewWithdrawalFields()
 
       $("input[name='unpublishing_reason_id']").change(this.revealCorrectForm)
       $('#unpublishing_redirect').change(this.hideExplanationIfRedirecting)
+      $("input[type=radio][name='use_previous_withdrawal']").change(this.revealNewWithdrawalFields)
```

Comment:
Not necessarily on this line. However we do need it in the `revealNewWithdrawalFields` function:

https://github.com/alphagov/whitehall/blob/65deb36c7055170eae02be1c59d7dc3c77c01b5e/app/assets/javascripts/admin/views/edition_workflow/confirm_unpublish.js#L48-L52

Because we go on to `.filter(':checked')` to find the checked radio.

However, if the Document does not have previous withdrawals, the input with `name="use_previous_withdrawal"` is actually `type="hidden"`. Although I note that you've commented on that too – so may be able to ditch both (see other comment thread).